### ISSUE TITLE
feature request:find distance btwn added devices

### ISSUE THREAD

**Issue by shanthiSreeDevaraju** [tananaev/traccar-manager-android#61]
it would be really helpful to find distance between devices. 
i want to implement this . but i need your help .

**Comment by tananaev** [comment]
What kind of help do you need? Also, please explain the use case.

**Comment by shanthiSreeDevaraju** [comment]
usecase: will be helpful to know if one of my friend at point A wants to reach other friend at point B . how much distance he has to travel and also time reqd(extra).

needed help:how do i start ?
(google maps already has this feature, so in traccar i can select two devices among list of added devices and get the distance(best and fastest route for now)  btwn two and time reqd( according to traffic).

**Comment by tananaev** [comment]
I think it would make sense to make it more generic. For example have a ruler on map to measure distances on map between any points.

Your question is too generic. If you need some help I would suggest to ask more specific questions.

**Comment by shanthiSreeDevaraju** [comment]
okay sorry I put too many tasks in that previous comment it seemed generic.
Can we add this feature on traccar?
image below:
https://inducesmile.com/android/android-find-distance-and-duration-between-two-points-on-android-map/

**Comment by shanthiSreeDevaraju** [comment]
![image](https://user-images.githubusercontent.com/31884031/35505022-b9c518ba-050a-11e8-9ab3-840dc795541f.png)

**Comment by tananaev** [comment]
I don't see why not.

**Comment by shanthiSreeDevaraju** [comment]
absolutely . can u please guide me how to .
because traccar manager android app is a web view of the traccar web interface. and also is using openstreetMap.
how to change it to use google maps api .where should i put my google api key , modify xml files etc.

**Comment by tananaev** [comment]
Well, I don't think you should be using Google Maps. Replying only on proprietary API is not a good idea.

**Comment by shanthiSreeDevaraju** [comment]
yes I thought so ,But using google maps API has an advantage in implementing this feature like:
The Google Maps Distance Matrix API returns information based on the recommended route between start and end points, as calculated by the Google Maps API, and consists of rows containing duration and distance values for each pair.
This service does not return detailed route information. Route information can be obtained by passing the desired single origin and destination to the Google Maps Directions API.

hence Anton sir , can u please guide me how to change openstreetMap to googleMaps API , where should i put my key etc
